test/test_configuration.rb

```diff
@@ -335,6 +335,18 @@ class TestConfiguration < JekyllUnitTest
         Jekyll.configuration(test_config.merge({ "config" => @paths[:other] }))
     end
 
+    should "load different config if specified with symbol key" do
+      allow(SafeYAML).to receive(:load_file).with(@paths[:default]).and_return({})
+      allow(SafeYAML)
+        .to receive(:load_file)
+        .with(@paths[:other])
+        .and_return({ "baseurl" => "http://wahoo.dev" })
+      allow($stdout).to receive(:puts).with("Configuration file: #{@paths[:other]}")
+      assert_equal \
+        site_configuration({ "baseurl" => "http://wahoo.dev" }),
```

Comment:
`.dev` is no longer a private extension, lets just make this example.com which is the IANA example domain that should be save (and is encouraged) as the test example.